Client/src/Components/Game/Users/PlayersList/usePlayersList.ts

```diff
@@ -0,0 +1,25 @@
+const UsePlayersList = () => {
```

Comment:
I think that the API calls should be in a parallel folder structure, because a call to get the current users in a room can be used also in other places, like while the room is waiting

Reply:
Maybe in this case we can use something like redux or a separate global state

Reply:
It will help, but don't you think it will be more organised if they will be seperated?

Reply:
not really, if its commonly used move it to a folder in Utils, something like 'UseUsers'

Reply:
ok